services/order.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth import get_user_model
+
```

Comment:
The blank line should be after `from django.db import transaction` import since it relates to the Django framework